### DISCUSSION TITLE
Filter Support On SnappyMail

### DISCUSSION THREAD

**Discussion by kshitijkurandwad211** [Mailu/Mailu#3385] [User support]
Hi,

I have a mailu server running and have SnappyMail as my webmail. As I can see in roundcube I can add mail filters in roundcube but this feature even though available on SnappyMail its not enabled probably because of SIEVE not enable on snappymail. Please help me how to add mail filters on snappymail

**Comment by nextgens** [comment]
If you go to ``/webmail/#/settings/filters`` you don't have something like this?
![image](https://github.com/user-attachments/assets/ee7d2328-fe17-414d-a52e-c9b05146a098)

**Comment by kshitijkurandwad211** [comment]
I do have, but SnappyMail provides UI filtering setup which is not showing here, anyway even after adding SIEVE script its not updating.

`require ["fileinto", "regex"];

if header :regex "Subject" "^WBR2\\. Investor Transactions for a Period$" {
    fileinto "CamsTransaction";
}`

Please check this

**Reply by nextgens** [reply]
"SnappyMail provides UI filtering setup"; I am not aware of any such thing. Can you provide a screenshot/reference?

"even after adding SIEVE script its not updating" here it works.

What do your logs tell you (journalctl -f while setting the filter) ? Which version of Mailu are you running?

**Reply by kshitijkurandwad211** [reply]
Give me some time as this was on server I changed in to roundcube and its working fine. Will set it up again and share the screen shots.

**Comment by kshitijkurandwad211** [comment]
Closing this discussion I started using roundcube
